### ISSUE TITLE
Crashes on Radeon RX580 with "VkPhysicalDeviceShaderFloat16Int8Features.shaderFloat16 not supported"

### ISSUE THREAD

**Issue by v-fox** [Vita3K/Vita3K#2602]
Updated to 20230524 / f88e3356e7b6a5be50730e795d2df596a1efa2e4 and got crash on startup with:
```
[23:09:52.620] |E| [debug_callback]: Validation layer: VkPhysicalDeviceShaderFloat16Int8Features.shaderFloat16 not supported (VK_ERROR_FEATURE_NOT_PRESENT)
[23:09:52.620] |E| [debug_callback]: Validation layer: terminator_CreateDevice: Failed in ICD /usr/lib64/libvulkan_radeon.so vkCreateDevice call
[23:09:52.620] |E| [debug_callback]: Validation layer: vkCreateDevice:  Failed to create device chain.
[23:09:52.620] |C| [operator()]: Unhandled C++ exception. vk::PhysicalDevice::createDevice: ErrorFeatureNotPresent
terminate called after throwing an instance of 'vk::FeatureNotPresentError'
  what():  vk::PhysicalDevice::createDevice: ErrorFeatureNotPresent
```

The likely culprit is edca8593aeab51bd4f05f118a4381981f4abe90d which forced FP16 on GPU that don't have it. I'm not sure if FSR actually requires it as [yuzu](https://github.com/yuzu-emu/yuzu) advertises it without crashing and Switch with GPU from 2015 supposedly natively has FSR (such as recent Legend of Zelda: TOTK).

**Comment by Macdu** [comment]
Should be fixed by #2603

**Comment by v-fox** [comment]
> Should be fixed by #2603

It starts now indeed. Too bad FSR is unavailable though.
